README.md

```diff
@@ -137,8 +138,10 @@ Login to NIH (https://portal.gdc.cancer.gov/) and download token under user prof
 `vim dcf_dataservice_settings`
 `vim creds.json`
 
-- Run job to set secrets for cloud and NIH tokens
-`bash ./dcf-dataservice/jobs/kube-setup-scripts.sh`
+### Final Setup
+Lastly to finish setting up tokens, we need to run `gen3 kubet-setup-data-replicate.sh` to refresh the secrets.
+Remember to pull from `cloud-automation` repo to ensure latest script is used.
+https://github.com/uc-cdis/cloud-automation/blob/master/gen3/bin/kube-setup-data-replicate.sh
```

Comment:
can you add something like run `gen3 kube-setup-data-replicate`?

Reply:
is line 142 good or is something else needed?